root/etc/e-smith/events/actions/fix-rsync-permissions

```diff
@@ -0,0 +1,141 @@
+fi
+
+if [[ -d "/var/lib/nethserver/nethcti/" ]]; then
+  /bin/chown -R asterisk:asterisk /var/lib/nethserver/nethcti/templates/customer_card
+  /bin/chown -R asterisk:asterisk /var/lib/nethserver/nethcti/static
+fi
+
+if [[ -d "/var/lib/nethserver/nethvoice/phonebook/uploads/" ]]; then
+  /bin/chgrp asterisk /var/lib/nethserver/nethvoice/phonebook/uploads
+  /bin/chown -R asterisk:asterisk /var/lib/nethserver/nethvoice/phonebook/uploads/*
+fi
+
+if [[ -d "/var/lib/nethserver/vmail/" && $(stat -c %U:%G "/var/lib/nethserver/vmail/") != "vmail:vmail" ]]; then
+  /bin/chown -R vmail:vmail /var/lib/nethserver/vmail
+  /bin/chgrp vmail /var/lib/nethserver/sieve-scripts
+  /bin/chown vmail:vmail /var/lib/nethserver/sieve-scripts/*.svbin
+fi
+
+if [[ -d "/var/lib/nethserver/imapsync/" ]]; then
+  /bin/chgrp -R vmail /var/lib/nethserver/imapsync
+fi
+
+if [[ -d "/var/lib/rspamd/" ]]; then
+  /bin/chown -R '_rspamd:_rspamd' /var/lib/rspamd
+  /bin/chown -R redis:redis /var/lib/redis
+  /bin/chown redis:mail /var/lib/redis/rspamd/dump.rdb
+fi
+
+if [[ -d "/var/lib/evebox/" ]]; then
+  /bin/chown -R suricata:suricata /var/lib/evebox
+fi
+
+if [[ -d "/var/lib/nethserver/webtop/" ]]; then
+  /bin/chown -R tomcat:tomcat /var/lib/nethserver/webtop
+  /bin/chown -R postgres:postgres /var/lib/nethserver/webtop/backup
+fi
+
+if [[ -d "/var/lib/nethserver/mattermost/" ]]; then
+  /bin/chown -R mattermost:mattermost /opt/mattermost
+  /bin/chown -R postgres:postgres /var/lib/nethserver/mattermost/backup
+  /bin/chown -R mattermost:mattermost /var/lib/nethserver/mattermost/data
+fi
+
+if [[ -d "/var/lib/nethserver/ejabberd/" ]]; then
+  /bin/chown -R ejabberd:ejabberd /var/lib/nethserver/ejabberd
+fi
+
+# if logs are included in the backup then fix their permissions too
+
+if [ "$(/sbin/e-smith/config getprop backup-data IncludeLogs)" == enabled ]; then
+  if [[ -d "/var/log/asterisk" ]]; then
+    /bin/chown -R asterisk:asterisk /var/log/asterisk
+  fi
+  
+  if [[ -d "/var/log/c-icap/" ]]; then
+    /bin/chown c-icap:c-icap /var/log/c-icap
+  fi
+  
+  if [[ -d "/var/log/clamav-unofficial-sigs/" ]]; then
+    /bin/chown -R clamupdate:clamupdate /var/log/clamav-unofficial-sigs
+  fi
+  
+  if [[ -d "/var/log/ejabberd/" ]]; then
+    /bin/chown -R ejabberd:ejabberd /var/log/ejabberd
+  fi
+  
+  if [[ -f "/var/log/freshclam.log" ]]; then
+    /bin/chown -R root:clamupdate /var/log/freshclam.log
+  fi
+  
+  if [[ -d "/var/log/imapsync/" ]]; then
+    /bin/chown -R vmail:vmail /var/log/imapsync
+  fi
+  
+  if [[ -d "/var/log/pbx/" ]]; then
+    /bin/chown -R asterisk:asterisk /var/log/pbx
+  fi
+  
+  if [[ -d "/var/log/redis/" ]]; then
+    /bin/chown -R redis:redis /var/log/redis
+    /bin/chown redis:asterisk /var/log/redis/nethvoice-report.log*
+    /bin/chown redis:mail /var/log/redis/rspamd.log*
+  fi
+  
+  if [[ -d "/var/log/rspamd/" ]]; then
+    /bin/chown -R _rspamd:_rspamd /var/log/rspamd
+  fi
+  
+  if [[ -d "/var/log/squid/" ]]; then
```

Comment:
Move to nethserver-squid

Reply:
Moved to [nethserver-squid#78](https://github.com/NethServer/nethserver-squid/pull/78)